html/janus.js

```diff
@@ -2134,15 +2101,15 @@ function Janus(gatewayCallbacks) {
 			var constraints = { mandatory: {}, optional: []};
 			pluginHandle.consentDialog(true);
 			var audioSupport = isAudioSendEnabled(media);
-			if(audioSupport === true && media != undefined && media != null) {
+			if(audioSupport && media) {
 				if(typeof media.audio === 'object') {
 					audioSupport = media.audio;
 				}
 			}
 			var videoSupport = isVideoSendEnabled(media);
-			if(videoSupport === true && media != undefined && media != null) {
-				var simulcast = callbacks.simulcast === true ? true : false;
-				var simulcast2 = callbacks.simulcast2 === true ? true : false;
+			if(videoSupport && media) {
+				var simulcast = (callbacks.simulcast === true);
+				var simulcast2 = (callbacks.simulcast2 === true);
 				if((simulcast || simulcast2) && !jsep && (media.video === undefined || media.video === false))
```

Comment:
Shouldn't the `(media.video === undefined || media.video === false)` become a `!media.video` too? Or did you leave it out on purpose?